connect-example/src/main/java/com/stripe/android/connect/example/MainActivity.kt

```diff
@@ -22,24 +21,17 @@ class MainActivity : ComponentActivity() {
         super.onCreate(savedInstanceState)
 
         setContent {
-            val viewModel = hiltViewModel<EmbeddedComponentLoaderViewModel>()
+            val viewModel = hiltViewModel<EmbeddedComponentLoaderViewModel>(this@MainActivity)
             val navController = rememberNavController()
             ConnectSdkExampleTheme {
                 NavHost(navController = navController, startDestination = MainDestination.ComponentPicker) {
                     composable(MainDestination.ComponentPicker) {
                         ComponentPickerContent(
                             viewModel = viewModel,
-                            openSettings = { navController.navigate(route = MainDestination.Settings) },
-                        )
-                    }
-                    composable(MainDestination.Settings) {
-                        val settingsViewModel = hiltViewModel<SettingsViewModel>()
-                        SettingsView(
-                            viewModel = settingsViewModel,
-                            onDismiss = { navController.safeNavigateUp() },
-                            onReloadRequested = viewModel::reload,
+                            openSettings = { navController.navigate(SettingsDestination.Settings) },
                         )
                     }
+                    settingsComposables(this@MainActivity, navController)
```

Comment:
👀 Refactored adding the Settings nav graph via util function `settingsComposables()`